LaTeX/LaTeX Log.sublime-syntax

```diff
@@ -5,21 +5,21 @@ scope: text.log.latex
 
 first_line_match: "This is (pdf|pdfe)?TeXk?, Version "
 
-contexts:
-  eol-pop:
-    - match: '$\n?'
-      pop: 1
+################################################################################
 
+contexts:
   main:
-    - include: invalids
-    - include: functions
-    - include: types
-    - include: tags
-    - include: comments
+    - include: log-entries
+    - include: file-paths
+    - include: doc-comments
+    - include: under-over-full
+    - include: strings
 
+  doc-comments:
     - match: ".*FiXme:"
       scope: meta.log.latex.fixme
```

Comment:
Despite the strange scopes being choosen by the original author, I'd say "FiXme" is another log-entry category - just like a warning. I may be wrong though.